src/main/java/hw_3/Task8.java

```diff
@@ -0,0 +1,27 @@
+package hw_3;
+
+public class Task8 {
+    public static void main(String[] args) {
+        /*Створіть рядок "Hello world!"
+                - Виведіть перший символ
+                - Виведіть останній символ
+                - Створіть масив символів, покладіть туди усі символи слова world
+                - Вивести масив у консоль
+                - Вивести в консоль передостанній елемент масиву, не використовуючи прямий доступ
+                за індексом (arr[3] – не згодиться)
+         */
+
+        String line = "Hello world!";
+        char firstChar = line.charAt(0);
+        char lastChar = line.charAt(line.length() - 1);
+
+        System.out.println("Перший символ: " + firstChar);
+        System.out.println("Останній символ: " + lastChar);
+
+        char[] worldArray = {'w', 'o', 'r', 'l', 'd'};
```

Comment:
1) You need a separate variable with "world" string value
2) You need to create a char[] and put your string there as a list of chars
3) You need to display you array
4) You need to display second last element for array

![image](https://github.com/AntBelyi/ALevel_automation/assets/66726924/916ae9a1-dbd0-4aa2-9eca-7debf5b1cfc9)

Reply:
fixed